docs/ota-client-guide/modules/ROOT/pages/garage-sign.adoc

```diff
@@ -0,0 +1,256 @@
+= garage-sign
+ifdef::env-github[]
+
+[NOTE]
+====
+We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
+====
+endif::[]
+
+xref:https://github.com/advancedtelematic/ota-tuf/tree/master/cli[garage-sign] is a tool for managing your OTA Connect software repository and securely signing software versions. It allows you to directly change, sign, and upload your xref:uptane.adoc[Uptane] metadata.
+
+.Why use `garage-sign`
+****
+`garage-sign` is useful in OTA Connect for managing the keys for signing your software. If your software image is not an OSTree image, you would need `garage-sign` to upload your xref:uptane.adoc[Uptane] metadata.
+
+Before using `garage-sign`, it is important to be familiar with the following two concepts.
+
+* *Offline keys*
++
+In a production environment, we recommend that you keep the keys for signing software versions offline. Therefore, the OTA Connect server will no longer sign software for you, instead, they will be signed locally by your machine. This maintains the security of the software because if someone gained access to your OTA Connect account, they will be able to send malicious software to your devices. However, if your keys are offline, your devices will not recognize the software as it will not have the correct credentials. This is where `garage-sign` becomes useful. For non-OSTree images, the `garage-sign` tool is used to manage and rotate your keys offline as well as sign your software.

```

Comment:
Some minor bits here, but most importantly: garage-sign also signs OSTree images. Is there something I'm missing about what you're trying to say, or can we just get rid of that specification? Here's my suggestion:
```suggestion
In a production environment, we recommend that you keep the keys for signing software versions offline. Therefore, the OTA Connect server will no longer sign software for you; instead, they will be signed locally by your machine. This maintains the security of the software because if someone gained access to your OTA Connect account, they will be able to send malicious software to your devices. However, if your keys are offline, your devices will not recognize the software as it will not have the correct credentials. This is where `garage-sign` becomes essential: it is necessary for managing and rotating your keys offline as well as signing your software.
```

Reply:
@patrickvacek applied your suggestion.